components/pages/privacy/consent/requests/generate-consent-request/index.jsx

```diff
@@ -48,7 +50,19 @@ export default function GenerateConsentRequest() {
       requestorInboxUrl: profile.inbox,
       options,
     });
-    setConsentId(vc.id);
+    if (vc) {
+      redirectToConsentManagementUi(
+        vc,
+        `https://localhost:3000//privacy/?signedVcUrl=${encodeURIComponent(
+          vc.id
+        )}`,
+        {
+          fallbackConsentManagementUi: `https://localhost:3000/privacy/consent/requests/${encodeURIComponent(
```

Comment:
Same here should `localhost` be removed?

Reply:
same as above